.github/workflows/release_gem.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    # Pattern matched against refs/tags
+    tags: ['**']
```

Comment:
I prefer
```suggestion
    tags:
      - '**'
```

Reply:
Changed. I don't have a strong preference on that one :)